### ISSUE TITLE
could not create Vfs.Dir from url. ignoring the exception and continuing

### ISSUE THREAD

**Issue by yb2020** [alibaba/nacos#7082]
**Describe the bug**
we rebuid nacos cluster at k8s, using nacos server 2.0.3。
at same time, we upgrade nacos-client 2.0.3
when we starting, we saw this error. 
```
WARN  org.reflections.Reflections [206] -| could not create Vfs.Dir from url. ignoring the exception and continuing
org.reflections.ReflectionsException: could not create Vfs.Dir from url, no matching UrlType was found [file:/opt/local/lib/libsvnjavahl-1.0.dylib]
```
but my mac have this file，This error report is misleading
<img width="466" alt="optlocal1ib" src="https://user-images.githubusercontent.com/46387614/137440419-c240303a-1ba1-40ee-b0cb-edced00c8c54.png">

the error report cause is Reflections scans directorys for "com.alibaba.nacos.naming.cluster.remote.request" ，i compare
code sources at "nacos-client-sources.2.0.3", but i can't find "com.alibaba.nacos.naming"，so this error has display.

![image](https://user-images.githubusercontent.com/46387614/137456267-630c6332-2b15-46e7-a70d-39c6febcc941.png)

in my project, i copy "com.alibaba.nacos.api.remote.PayloadRegistry" and remove "com.alibaba.nacos.naming" package scans , the program has startuped at right way.

<img width="1014" alt="Pasted Graphic 2" src="https://user-images.githubusercontent.com/46387614/137441217-e7e7e33c-c0f6-4cba-85cf-cec3bb0d85ef.png">

at least, nacos-client add this package to scan,  the right solution is not to remove it.
and i find package in github project of nacos.

![image](https://user-images.githubusercontent.com/46387614/137441426-ec82a8de-26ae-447d-a946-131ad8308236.png)

why nacos-client-2.0.3 not add package "com.alibaba.nacos.naming" at jar?

**Desktop (please complete the following information):**
 - OS: mac os big sur
 - Version nacos-client 2.0.3
 - Module naming/config
 - SDK com.alibaba.nacos.nacos-client:2.0.3/sprinboot 2.3.8.RELEASE/springcloud Hoxton.SR9

**Additional context**

**Comment by wangzhen1816** [comment]
I also met , who can help answer

**Comment by cyberword** [comment]
Same questions,  Reflections will scan this package but it not exist, It may BLOCKED when some file is incorrect

**Comment by samuel072** [comment]
I also met , who can help answer

**Comment by samuel072** [comment]
这么久都不解决哈哈吗 1.4.2的版本也有问题，会自动断开，说token过期。服了

**Comment by onewe** [comment]
I guess it has to create the request and response at the api-module:

- create a package `com.alibaba.nacos.api.naming.cluster.remote.request`
- create the request and response in this package

**Comment by onewe** [comment]
If I'm right I will resolve it 😂

**Comment by farhad663** [comment]
Alright

Sent from Yahoo Mail on Android 
 
  On Tue, 11 Jan 2022 at 12:15, ***@***.***> wrote:   


If I'm right I will resolve it 😂

**Comment by onewe** [comment]
If I will do that like this way, the community won't agree it. because the api-module will depend on the core-module.😂emmmm

**Comment by onewe** [comment]
@li-xiao-shuang @realJackSun @brotherlu-xcq What do all you think this issue?

**Comment by li-guohao** [comment]
same questions

**Comment by xThink** [comment]
same question on ubuntu server when i deployed.

**Comment by lszall** [comment]
same questions

**Comment by onewe** [comment]
Please update to latest version,  reflections package had been removed already.

**Comment by KomachiSion** [comment]
Fixed by 2.1.0
